teslajsonpy/sentry_mode.py

```diff
@@ -43,11 +44,21 @@ def __init__(self, data, controller):
         last_update = self._controller.get_last_update_time(self._id)
         if last_update >= self.__manual_update_time:
             data = self._controller.get_state_params(self._id)
-            self.__sentry_mode = data and data["sentry_mode"]
+            self.__sentry_mode_available = data and data["sentry_mode_available"]
+            if self.__sentry_mode_available:
+                self.__sentry_mode = data["sentry_mode"]
+
+    def is_available(self):
+        """Return whether the sentry mode is available."""
+        return self.__sentry_mode_available
+
+    def is_enabled(self):
+        """Return whether the sentry mode is enabled, always False if sentry mode is not available."""
```

Comment:
HA wants you to return `None` typically for values, but I'm not sure for switches. Can you test HA with `None` to see what the behavior is for a switch?

Reply:
OK, I will test this.

Reply:
A None value behaves as a False. Change to `available` and `is_on as per HA`entity.